browser/application.py

```diff
@@ -80,6 +80,18 @@ def create_app(**config_overrides):
 
     compress.init_app(app)
 
+    # For flask monitoring dashboard
+    dashboard.config.init_from(file='fmd_config.cfg')
```

Comment:
Does this config file exist?

Reply:
This config file is in my local copy of the repo, but not tracked, similar to .env. The config file contains database credentials and dashboard credentials here are the non-sensitive contents of the config file I've been using:

```
[dashboard]
VERSION=refactor
GIT=../.git/
CUSTOM_LINK=dashboard
MONITOR_LEVEL=3
OUTLIER_DETECTION_CONSTANT=2.5
SAMPLING_PERIOD=20
ENABLE_LOGGING=True

[visualization]
TIMEZONE=US/Pacific
```

The sensitive fields (with sensitive values removed) are:
```
[authentication]
USERNAME=
PASSWORD=
SECURITY_TOKEN=

[database]
TABLE_PREFIX=fmd
DATABASE=
```

Reply:
Is it a problem if the file does not exist? Should include a default one without the sensitive fields?

Reply:
I've added the an example config file with all the info above, except for a blank version name

Reply:
I think we should refactor the filename to [be an environment variable](https://flask-monitoringdashboard.readthedocs.io/en/v1.10.5/configuration.html), and maybe we can use the above example file with the defaults filled in?

`dashboard.config.init_from(envvar='DASHBOARD_CONFIG')`

Reply:
I've updated .env.example to include `DASHBOARD_CONFIG=fmd_config.cfg.example` and added the above code line to application.py. It looks like things are working out of the box. With the USERNAME and PASSWORD fields blank, the dashboard defaults to admin/admin credentials, and with DATABASE blank, it defaults to a SQLite database flask_monitoringdashboard.db.

Reply:
Instead of using the `env_var` arg, what do you think about adding `DASHBOARD_CONFIG` to `config.py`?

Also, do we want to always enable it or can we toggle whether to use it? Maybe if `DASHBOARD_CONFIG` is empty we don't need to initialize the dashboard?

Reply:
I like that behavior. As an empty string is falsy, I wrapped the dashboard initialization in application.py in an `if config.DASHBOARD_CONFIG` block

Reply:
beautiful!

Reply:
Last thing I can think of - Should we get rid of the `.example` extension on the configuration file?

Reply:
I buy remove the .example extension for the dashboard config. I included it because we also included it for .env.example, but I suppose that .env.example is nearly empty while the dashboard config is nearly complete, other than the authentication info. I'm going to remove the .example from the dashboard config file and add a comment inside it to highlight the default behavior of the empty fields.

Reply:
Hmmmm, I just realized that if we remove the `.example` from the dashboard config file, then git will be aware of changes to it and prompt us to track the changes, which would often include sensitive info like credentials. By explicitly including the `.example` extension, the user can more easily create an untracked dashboard config file within their repo by removing `.example`.

Reply:
I'm convinced.